### PR TITLE
SVG assets

### PR DESCRIPTION
**Pull request by HighlanderCZ**
# SVG assets instead of blurry PNGs

## Pull Request Type

-  [ ] Bugfix
-  [ ] Feature Implementation
-  [ ] Documentation
-  [x] Other

## Related issue
#2007 

## Description

Changes icon and logo in the top bar for all themes to use SVG assets instead of PNGs.

## Screenshots 
comparison

## Testing 
The app runs correctly in dev environment and builds without errors.

## Desktop

-  **OS:** macOS
-  **OS Version:** Sonoma 14.4
-  **FreeTube version:** v0.20.0 Beta

### CODE REVIEW

**Comment by PikachuEXE** [comment]
How do you create the SVG files?

For reviewers: You have to click on `View file` on removed PNG files to see them, annoying

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
would this solve https://github.com/FreeTubeApp/FreeTube/issues/2007 ?

**Comment by HighlanderCZ** [comment]
No, I didn't touch this particular icon

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Could you also address that one?

**Comment by HighlanderCZ** [comment]
> Could you also address that one?

It didn't seem jagged on my display, but nevertheless, I recreated the Windows icon in a higher res version -- from 1024x1024 all the way down to 16x16. Can you download the icon itself and check if it displays better for you?

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — DISMISSED** [review]
Looks very clean!

**Review by jasonhenriquez — APPROVED** [review]
LGTM

File: src/main/index.js [hunk 1]
```diff
@@ -512,9 +512,9 @@ function runApp() {
       backgroundColor: windowBackground,
       darkTheme: nativeTheme.shouldUseDarkColors,
       icon: process.env.NODE_ENV === 'development'
-        ? path.join(__dirname, '../../_icons/iconColor.png')
+        ? path.join(__dirname, '../../_icons/iconColor.svg')
         /* eslint-disable-next-line n/no-path-concat */
-        : `${__dirname}/_icons/iconColor.png`,
+        : `${__dirname}/_icons/iconColor.svg`,
```
Comment (absidue) [inline src/main/index.js:517]: This file needs to stay as a png, Electron doesn't support using svgs as window icons.

Comment (jasonhenriquez) [reply src/main/index.js:517]: [Source here](https://github.com/electron/electron/issues/9642). Good catch!

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Review by PikachuEXE — APPROVED** [review]

**Review by absidue — APPROVED** [review]

**Pull request merged**